cmd/clusterctl/cmd/config_repositories_test.go

```diff
@@ -193,3 +202,28 @@ var expectedOutputYaml = `- File: core_components.yaml
   ProviderType: InfrastructureProvider
   URL: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/releases/latest/
 `
+var expectedOutputName = `cluster-api
```

Comment:
Maintaining this list is a continuous effort. Is it possible to revisit this in a way that we check the output is in the expected format, vs checking the values itself?

Reply:
I really would like to see this (revisit in order to reduce maintenance effort) addressed before merging...